hathor/merged_mining/coordinator.py

```diff
@@ -1307,7 +1307,7 @@ async def _update_hathor_block(self, mining: IMiningChannel) -> None:
             # self.log.debug('hathor.get_block_template response', block=block, height=height)
             self.hathor_coord_job = HathorCoordJob(block, height)
             self.log.debug('new Hathor block template', height=height, weight=block.weight)
-            self.update_merged_block()
+            await self.update_merged_block()
```

Comment:
Why was this needed?

Reply:
Oh, this is actually a bugfix that the updated `mypy` caught.

Reply:
The bug itself is pretty minor and does not affect any part of the fullnode itself. And also there shouldn't be anyone using our merged mining coordinator, so the impact is very minimal.